HedgeModManager/Languages/Languages.xaml

```diff
@@ -6,6 +6,7 @@
     <local:LanguageList x:Key="Languages">
         <local:LangEntry FileName="en-AU" Name="English (Australia)"/>
         <local:LangEntry FileName="en-US" Name="English (United States)"/>
+        <local:LangEntry FileName="uk-UA" Name="Українська (Ukrainian)"/>
```

Comment:
Just notice a minor issue here. Within the brackets should be the name of the region. In the case of uk-UA, it should be Ukraine as I have suggested below.

```suggestion
        <local:LangEntry FileName="uk-UA" Name="Українська (Ukraine)"/>
```